clients/python/dd_bench.py

```diff
@@ -37,6 +37,7 @@
 parser.add_argument('--img-width',help='image width',type=int,default=224)
 parser.add_argument('--img-height',help='image height',type=int,default=224)
 parser.add_argument('--bw',help='whether images are bw',action='store_true')
+parser.add_argument('--histogram-equalization',help='whether we apply an histogram equalization to images', action='store_true')
```

Comment:
Suggestion, maybe abbreviated as `--eqhist`, unless it's considered bad practice!

Reply:
For API, self explicit name if often better.
For cli tools, it always good to have the explicit option (for usage of the cli in script) and a short alias (for usage in interactive terminal)